test/boost/sstable_compaction_test.cc

```diff
@@ -215,7 +215,7 @@ SEASTAR_TEST_CASE(compact) {
             auto new_sstable = [&env, gen = make_lw_shared<unsigned>(generation), s] {
                 return env.make_sstable(s, (*gen)++);
             };
-            return compact_sstables(sstables::compaction_descriptor(std::move(sstables), default_priority_class()), cf, new_sstable).then([&env, s, cf] (auto) {
+            return compact_sstables(sstables::compaction_descriptor(std::move(sstables), default_priority_class()), cf, new_sstable).then([&env, s, cf, generation] (auto) {
```

Comment:
might want to drop this commit, as Benny has given an overhaul to these tests.

Reply:
I'll rebase.